test-clients/src/main/java/com/hedera/services/yahcli/config/ConfigManager.java

```diff
@@ -83,6 +83,12 @@ public Map<String, String> asSpecConfig() {
         return specConfig;
     }
 
+    public boolean allowListExist(long account) {
```

Comment:
This needs a better name, or maybe a clearer function. It returns true if the list is empty or if it contains the given account. Is it supposed to return false if the list is empty?

Reply:
no it has to return true and I think I changed the name base on @mhess-swl proposal

Reply:
I didn't think about the name itself, just the spelling/grammar. A more descriptive name, or a comment explaining what is allowed, would be nice.

Reply:
ok I just implemented base on the issue what was there

Reply:
done fixed